gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/configuration/GLDSimulationOutputConfigurationHandler.java

```diff
@@ -343,7 +343,7 @@ void parseMeasurement(Map<String, JsonArray> measurements, JsonObject measuremen
 				propertyName = "voltage_" + phases;
 			} else if (measurementType.equals("Pos")) {
 				objectName = conductingEquipmentName;
-				propertyName = "status";
+				propertyName = "phase_" + phases + "_state";
```

Comment:
Why are these changes here ? Update from upstream develop and update pull request.